features/import_videos.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Importing videos
```

Comment:
This feature has already been written in tests, save for importing via a YouTube link.